apps/daffio/src/app/core/sidebar/registration/type.ts

```diff
@@ -1,12 +1,21 @@
-import { DaffSidebarRegistration } from '@daffodil/design/sidebar';
+import {
+  DaffSidebarMode,
+  DaffSidebarRegistration,
+} from '@daffodil/design/sidebar';
+
+/**
+ * A strategy that determines whether a section of the sidebar should be visible based on the screen breakpoint and sidebar mode.
+ */
+export type DaffioSidebarSectionStrategy = (isBigTablet: boolean, mode: DaffSidebarMode) => boolean;
```

Comment:
@xelaint is the breakpoint (big tablet or not) and the current mode sufficiently flexible for your use case?

Reply:
I'm a bit confused by how the code works. How would I define a footer that should be shown regardless of screen size vs a footer that should only be shown on mobile view?

Reply:
I've added some prebuilt strategies, see `daffioSidebarOnlyShowOnMobileSectionStrategy`. Let me know if you need more complex logic or prebuilt strategies

Reply:
How do I use `daffioSidebarOnlyShowOnMobileSectionStrategy`?

Reply:
`footerStrategy: daffioSidebarOnlyShowOnMobileSectionStrategy` in the sidebar registration